TRB_label_assist/clustering.py

```diff
@@ -101,15 +102,17 @@ def add_loc_clusters(
         for r in radii:
             # this is using a modified Similarity class that bins start/end
             # points separately before creating trip-level bins
-            sim_model = eamts.Similarity(loc_df,
-                                         radius_start=r,
-                                         radius_end=r,
-                                         shouldFilter=False,
-                                         cutoff=False)
-            # we only bin the loc_type points to speed up the alg. avoid
-            # unnecessary binning since this is really slow
-            sim_model.bin_helper(loc_type=loc_type)
-            labels = sim_model.data_df[loc_type + '_bin'].to_list()
+
+            model_config = {
+                "metric": "od_similarity",
+                "similarity_threshold_meters": r,  # meters,
+                "apply_cutoff": False,
+                "incremental_evaluation": False
+            }    
+
+            sim_model = eamtg.GreedySimilarityBinning(model_config)
+            sim_model.fit(loc_df)   # TODO: Change the type of data being passed to 
+            labels = sim_model.tripLabel
```

Comment:
why isn't this just `labels = sim_model.keys()`

Reply:
do you mean `sim_model.bins.keys()`? 

`sim_model.bins.keys()` would return just the ids of the bins.  For e.g.  0 , 1 ,2  and so on. That too in any order. What we need is a trip and binNo. mapping, For e.g. like this

Trip 1   -->    bin No. (say 2)
Trip 2.  -->    bin No. (say 3) 
Trip 3   -->   bin No.( say 2)
Trip 4   -->   bin No.( say 1)
Trip 5   -->   bin No.( say 1)
.
.
.

Reply:
I wrote this before https://github.com/e-mission/e-mission-server/pull/930#discussion_r1291765346